scripts/workspace-conversion/README.md

```diff
@@ -21,41 +21,47 @@ The request made to retrieve the workspace details is an XHR request to rawls to
-| 7 | Cohort Name | `library:datasetName` | `dct:title` |
-| 8 | Dataset Version | `library:datasetVersion` | `dct:version` |
-| 9 | Cohort Description | `library:datasetDescription` | `dct:description` |
-| 10 | Dataset Owner | `library:datasetOwner`<br>ex: "NCI" | `TerraDCAT_ap:hasOwner`<br>`TerraDCAT_ap:hasDataCollection.0.dct:identifier`<br>If we have enough information, we can also build:<br> `TerraDCAT_AP:hasDataCollection.0.dct:publisher: National Cancer Institute`<br>`TerraDCAT_AP:hasDataCollection.0.dct:title: National Cancer Institute` |
-| 11 | Dataset Custodian | `library:datasetCustodian`<br>ex: "dbGAP" | `TerraDCAT_ap:hasCustodian`<br>`TerraDCAT_ap:hasDataCollection.0.dct:identifier`<br>Note: This conflicts with "Dataset Owner", but I dont think we have another option for where to put this. |
-| 12 | Dataset Depositor | `library:datasetDepositor` | `contributors.0.contactName`<br>`contributors.0.correspondingContributor = true` |
-| 13 | Contact Email | `library:contactEmail` | `contributors.0.email` |
-| 14 | Research Institute | `library:institute` | `contributors.0.institution`<br>`prov:wasAssociatedWith` |
-| 15 | Primary Disease Site | `library:primaryDiseaseSite`<br>ex: "Pleura" | `samples.disease.0`?<br>ex: "Brain Cancer" |
-| 16 | Project Name | `library:projectName`<br>ex: "TCGA" | `prov:wasGeneratedBy.0.TerraCore:hasAssayType.0` |
-| 17 | Genome Reference Version | `library:reference`<br>ex: "GRCh37/hg19" |  |
-| 18 | Data File Formats | `library:dataFileFormats`<br>ex: "TXT, MAF" | `files.0.dcat:mediaType`<br>`files.0.count = 0`<br>`files.0.byteSize = 0`<br>Note: No way of knowing how many files match each file format |
-| 19 | Profiling Instrument Type | `library:technology` |  |
-| 20 | Profiling Protocol | `library:profilingProtocol` |  |
-| 21 | Depth of Sequencing Coverage (Average) | `library:coverage` |  |
-| 22 | Study Design | `library:studyDesign`<br>ex: "Tumor/Normal" | `samples.disease.0`<br>Note: concat value with "Primary Disease Site"? |
-| 23 | Cell Type | `library:cellType`<br>ex: "Primary tumor cell, Whole blood" | `samples.disease.0`<br>Note: concat value with "Primary Disease Site"? |
-| 24 | Reported Ethnicity | `library:ethnicity` |  |
-| 25 | Cohort Country of Origin | `library:cohortCountry`<br>ex: "USA" |  |
-| 26 | Requires External Approval | `library:requiresExternalApproval` |  |
-| 27 | library Metadata Schema Version Number | `library:lmsvn` |  |
-| 28 | Structured Data Use Limitations Version Number | `library:dulvn` |  |
-| 29 | ORSP Conset Code | `library:orsp` | `TerraDCAT_ap:hasConsentGroup` |
+| i   | Attribute                                      | Rawls Workspace Path | Data Catalog Entry Path |
+|-----|------------------------------------------------| -------------------- | -------------------------- |
```

Comment:
This is an auto-reformat by IntelliJ, which happened once I updated the header line for `i` to have three hyphens, which is what the markdown spec requires for a column header row.